community/examples/hpc-build-slurm-image.yaml

```diff
@@ -117,3 +117,4 @@ deployment_groups:
     settings:
       machine_type: n2d-standard-4
       instance_image: $(vars.built_instance_image)
+      enable_devel: false
```

Comment:
Add a comment to this line, something like:
`# Will cause Slurm auto-scaling scripts to be sourced from built image

Reply:
Done